src/components/SelectComponents/Select/__tests__/Select.spec.js

```diff
@@ -54,25 +51,72 @@ describe('Select', () => {
             clickWrapper();
             expect(wrapper.find('li')).toHaveLength(SUGGESTIONS.length);
         });
-        it('should toggle list when arrow element is clicked', () => {
+        it('should open list when arrow element is clicked', () => {
             // originally to be closed
             expect(wrapper.find('li')).toHaveLength(0);
 
             // open items list
-            wrapper.find('.Select__dropdownIcon').at(0).simulate('click');
+            wrapper.find('svg').at(0).simulate('click');
             expect(wrapper.find('li')).toHaveLength(SUGGESTIONS.length);
 
-            // close items list
-            wrapper.find('.Select__dropdownIcon').at(0).simulate('click');
+            // select item
```

Comment:
why this change? I think it is a good idea to see that clicking on the icon works. Also checking that selection closes the dropdown is a different test then implied in the description, and you have it (see 3 test below)

Reply:
Actually, the arrow has the `pointer-events: none` style and it is not testable. For some reason, in the test environment, it opens when it was clicked, but it doesn't close.

Reply:
In that case, perhaps better to comment out the original part, add a comment to fix it. Now it is a different test then what we want to do, and it is not clear unless someone looks into the details. So changes can be more easily missed. 

Should we consider moving interaction tests to Storybook? (maybe can think about it separately)

Reply:
I commented out the original test and also added `TODO` description. I think it is a nice idea to have integration tests.